src/analysis/exploration/printRxnFormula.m

```diff
@@ -75,15 +75,15 @@
 
 parser = inputParser();
 parser.addRequired('model',@isstruct) % we only check, whether its a struct, no details for speed
-parser.addParamValue('rxnAbbrList',model.rxns,@(x) iscell(x) || ischar(x))
-parser.addParamValue('printFlag',true,@(x) isnumeric(x) || islogical(x))
-parser.addParamValue('lineChangeFlag',true,@(x) isnumeric(x) || islogical(x));
-parser.addParamValue('metNameFlag',false,@(x) isnumeric(x) || islogical(x));
-parser.addParamValue('fid',1, @isnumeric);
-parser.addParamValue('directionFlag',false,@(x) isnumeric(x) || islogical(x));
-parser.addParamValue('gprFlag',false,@(x) isnumeric(x) || islogical(x));
-parser.addParamValue('proteinFlag',false,@(x) isnumeric(x) || islogical(x));
-parser.addParamValue('printBounds',false,@(x) isnumeric(x) || islogical(x));
+parser.addParameter('rxnAbbrList',model.rxns,@(x) iscell(x) || ischar(x))
```

Comment:
May you leave here `addParamValue` in order to keep the compatibility with MATLAB versions older than `R2013b`? Thanks.